### PR TITLE
[MWPW-158021] Move Nala tests to Milo Repo

### PR DESCRIPTION
**Pull request by skumar09**
1.  Move following Nala tests to Milo Repo

### Blocks
- Accordion 
- Aside
- Action-item
- Carousel
- Card
- Chart
- Columns
- Figure
- HowTo
- Icon
- IFrame
- Marketo
- Marquee
- Media
- Merchcard
- Modal
- Quote
- Review
- Table
- Tabs
- Text
- Video
- Zpattern

### Features
- Georouting
- Commerce
- Feds
- OST
- Promotions

2. Set Default GitHub Action for running Nala Tests on PRs

Resolves: [MWPW-158021](https://jira.corp.adobe.com/browse/MWPW-158021)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://nala-tests--milo--skumar09.hlx.page/?martech=off

### CODE REVIEW

**Review by mokimo — APPROVED** [review]
Great! Lovely to see more nala tests moving in

File: nala/features/osttools/ost.spec.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        offerType: 'TRIAL',
+        price: 'US$263.88',
+        opticalPrice: 'US$21.99',
+        term: '/yr',
+        opticalTerm: '/mo',
+        unit: 'per license',
+        taxLabel: 'excl. tax',
+      },
+      browserParams: '?token=',
+      tags: '@ost @commerce @regression @nopr @jk',
```
[the remark this answers is not in the document]
Comment (jk) [reply nala/features/osttools/ost.spec.js:22]: @afmicka make sure to escape GitHub handles in your comments to prevent unintended notifications

**Comment by afmicka** [comment]
@skumar09 OST tests are not being executed under commerce nor ost tag. It does not find the tests at all. Do you know why? The other commerce tests are executing  OK with the tags.

`npm run nala stage @ost`

Screenshot 2024-09-09 at 16 07 31

**Review by afmicka — APPROVED** [review]

**Review by JackySun9 — APPROVED** [review]

**Review by Dli3 — APPROVED** [review]

**Comment by JackySun9** [comment]
@skumar09 please fix this

**Review by robert-bogos — APPROVED** [review]

**Review by TsayAdobe — CHANGES_REQUESTED** [review]
Please add the nala directory to`.hlxignore`

**Review by TsayAdobe — APPROVED** [review]

**Pull request merged**